replica/distributed_loader.cc

```diff
@@ -304,51 +304,51 @@ future<uint64_t> reshape(sstables::sstable_directory& dir, replica::table& table
     uint64_t reshaped_size = 0;
 
     while (true) {
-            auto reshape_candidates = boost::copy_range<std::vector<sstables::shared_sstable>>(dir.get_unshared_local_sstables()
-                    | boost::adaptors::filtered([&filter] (const auto& sst) {
-                return filter(sst);
-            }));
-            auto desc = table.get_compaction_strategy().get_reshaping_job(std::move(reshape_candidates), table.schema(), iop, mode);
-            if (desc.sstables.empty()) {
-                break;
-            }
+        auto reshape_candidates = boost::copy_range<std::vector<sstables::shared_sstable>>(dir.get_unshared_local_sstables()
```

Comment:
nit, in the title of commit message: s/Iindentation/indentation/

Reply:
Thanks, fixed